### ISSUE TITLE
Request for training code

### ISSUE THREAD

**Issue by bluestyle97** [Yeh-yu-hsuan/BiFuse#7]
Hi, I want to train BiFuse model on a new dataset, but this repo contains no training code, such as loss function, metrics and training script, etc. So could you please kindly make the training code public? Thanks a lot.

**Comment by kireoxdz** [comment]
Hi, I want to train BiFuse model on a new dataset, too. could you please kindly make the training code public?

**Comment by gordonx8** [comment]
Hi, I also want to retrain BiFuse using new dataset, could you please kindly share the training code?
